Mathlib/MeasureTheory/Measure/Support.lean

```diff
@@ -0,0 +1,241 @@
+/-
+Copyright (c) 2025 Jon Bannon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jon Bannon, Jireh Loreaux
+-/
+
+import Mathlib.MeasureTheory.Measure.MeasureSpace
+import Mathlib.MeasureTheory.Measure.OpenPos
+import Mathlib.MeasureTheory.Measure.MeasureSpaceDef
+import Mathlib.Topology.Defs.Filter
+import Mathlib.Order.Filter.SmallSets
+
+/-!
+# Support of a Measure
+
+This file develops the theory of the **support** of a measure `μ` on a
+topological measurable space. The support is defined as the set of points whose every open
+neighborhood has positive measure. We give equivalent characterizations, prove basic
+measure-theoretic properties, and study interactions with sums, restrictions, and
+absolute continuity. Under various Lindelöf conditions, the support is conull,
+and various descriptions of the complement of the support are provided.
+
+## Main definitions
+
+* `Measure.support` : the support of a measure `μ`, defined as
+  `{x | ∃ᶠ u in (𝓝 x).smallSets, 0 < μ u}` — equivalently, every neighborhood of `x`
+  has positive `μ`-measure.
+
+## Main results
+
+* `compl_support_eq_sUnion` and `support_eq_sInter` : the complement of the support is the
+  union of open measure-zero sets, and the support is the intersection of closed sets whose
+  complements have measure zero.
+* `isClosed_support` : the support is a closed set.
+* `support_mem_ae_of_isLindelof` and `support_mem_ae` : under Lindelöf (or hereditarily
+  Lindelöf) hypotheses, the support is conull.
+
+## Notation
+
+* `μ.support` : the support of measure `μ`.
+* `(𝓝 x).smallSets` : the frequently-small-sets filter used in the filter-theoretic definition.
+
+## Tags
+
+measure theory, support, filter, Lindelöf, hereditarily Lindelöf,
+absolute continuity, restriction, sum of measures, null measurable, conull
+-/
+
+section Support
+
+namespace MeasureTheory
+
+namespace Measure
+
+open scoped Topology
+
+variable {X : Type*} [MeasurableSpace X]
+
+lemma AbsolutelyContinuous.null_mono {μ ν : Measure X} (hμν : μ ≪ ν) ⦃t : Set X⦄
+    (ht : ν t = 0) : μ t = 0 :=
+  hμν ht
+
+lemma AbsolutelyContinuous.pos_mono {μ ν : Measure X} (hμν : μ ≪ ν) ⦃t : Set X⦄
+    (ht : 0 < μ t) : 0 < ν t := by
+  contrapose! ht
+  simp_all [hμν.null_mono]
```

Comment:
These don't belong here. They should go in the file which defines absolute continuity of measures.